### PR TITLE
Site optimisation

### PR DESCRIPTION
**Pull request by owner6**

### CODE REVIEW

File: html/1woodEnergyConverter.html [hunk 1]
```diff
@@ -50,66 +50,45 @@ <h2 class="text-gray">Вычисления</h1>
        <div>
           <div class="panel">
             <h3>Калькулятор делает вычисления</h3>
-          
-          
-              <p>
               <label> Введите колличество топлива</label>
               <input id="inputFuel" type="number" placeholder= "Ресурсы"
                oninput ="FuelEnergyConverter(this.value)" onchange="FuelEnergyConverter(this.value)">
               </p>
```
Comment (Dever4eg) [inline html/1woodEnergyConverter.html:56]: Did you forget to remove `</p>` ?

File: html/1woodEnergyConverter.html [hunk 2]
```diff
@@ -50,66 +50,45 @@ <h2 class="text-gray">Вычисления</h1>
...
-                            </div>
-                            <button onclick="know()" class="">Узнайте</button>
-                      
-                      <span id="result"></span>
-                  </div>
-                </div>
-             </div>
+              <h3>Когда рубить дерево</h3>
+              <label for="num">Укажите дерево которое нашли (м):</label> 
+              <input type="text" class="" id="num">
```
Comment (Dever4eg) [inline html/1woodEnergyConverter.html:67]: Why did you add an empty class attribute?

File: html/1woodEnergyConverter.html [hunk 3]
```diff
@@ -50,66 +50,45 @@ <h2 class="text-gray">Вычисления</h1>
...
+            
+              <button onclick="know()" class="">Узнайте</button>
```
Comment (Dever4eg) [inline html/1woodEnergyConverter.html:69]: It it better to add button click handler in the js code, not in the html

File: html/1woodEnergyConverter.html [hunk 4]
```diff
@@ -50,66 +50,45 @@ <h2 class="text-gray">Вычисления</h1>
...
+              <p><span id="result"></span></p>
```
Comment (Dever4eg) [inline html/1woodEnergyConverter.html:70]: Same as before, you can change id to **result**Something

File: html/1woodEnergyConverter.html [hunk 5]
```diff
@@ -50,66 +50,45 @@ <h2 class="text-gray">Вычисления</h1>
...
             </div>
+           
+           
+            <br>
+            <div class="panel">
+              <h3>Итоговая оценка студента</h3>
+              <label for="exam">Сколько балов вы набрали:</label> 
+              <input type="text1" class="" id="exam">
```
Comment (Dever4eg) [inline html/1woodEnergyConverter.html:78]: Empty class again

Comment (Dever4eg) [inline html/1woodEnergyConverter.html:78]: What is `text1`?

File: html/1woodEnergyConverter.html [hunk 6]
```diff
@@ -50,66 +50,45 @@ <h2 class="text-gray">Вычисления</h1>
...
+              
+              <label for="projects">Сколько проэктов вы сделали:</label>   
+              <input type="text1" class="" id="projects">
+            
+              <button onclick="studentScores()" class="">Узнайте</button>
+              <p><span id="result1"></span></p>
```
Comment (Dever4eg) [inline html/1woodEnergyConverter.html:84]: `result1` - is not a good name

File: html/1woodEnergyConverter.html [hunk 7]
```diff
@@ -50,66 +50,45 @@ <h2 class="text-gray">Вычисления</h1>
...
             </div>
-
-
-          
-          
-
+            <br>
+            <p>game over</p> 
           
+
```
Comment (Dever4eg) [inline html/1woodEnergyConverter.html:89]: Unnecessary empty line

File: html/contacts.html [hunk 8]
```diff
@@ -53,4 +53,5 @@ <h2 class="text-gray">Контакты нашего проекта<h2>
     
 
 
-    </body>
\ No newline at end of file
+    </body>
+</html>
```
Comment (Dever4eg) [inline html/contacts.html:57]: No newline in the end of file

File: html/help.html [hunk 9]
```diff
@@ -52,4 +52,5 @@ <h2 class="text-gray">Вы можете задать любой интересу
       </main>
 
 
-</body>
\ No newline at end of file
+</body>
+</html>
```
Comment (Dever4eg) [inline html/help.html:56]: No newline in the end of file

File: scrypt.js [hunk 10]
```diff
@@ -30,4 +30,31 @@ function know(){
   else {
     s.innerHTML = 'Нету дерева';
   }
-}
\ No newline at end of file
+}
+
+
+function studentScores(){
+  var e,p,r;
```
Comment (Dever4eg) [inline scrypt.js:37]: Better to use `const` or `let` instead of `var`

File: scrypt.js [hunk 11]
```diff
@@ -30,4 +30,31 @@ function know(){
...
+  e = document.getElementById('exam').value;
+  p = document.getElementById('projects').value;
+  r = document.getElementById('result1');
+  
+  e = parseInt(e);
```
Comment (Dever4eg) [inline scrypt.js:42]: Bad variables name. Better to use some words that describe these vars

File: scrypt.js [hunk 12]
```diff
@@ -30,4 +30,31 @@ function know(){
...
+  else if (e>50 && p>=2) {
+    r.innerHTML = 'Вы набрали 75 балов';
+  } 
+  else {
+    r.innerHTML = 'Вы набрали 0 балов';
+  }
+}
+
+
+
```
Comment (Dever4eg) [inline scrypt.js:60]: Three empty lines instead of one

**Review by Dever4eg — CHANGES_REQUESTED** [review]

File: html/1woodEnergyConverter.html (hunk 1 above)
Comment (owner6) [reply html/1woodEnergyConverter.html:56]: removed "</p>"

File: html/1woodEnergyConverter.html (hunk 2 above)
Comment (owner6) [reply html/1woodEnergyConverter.html:67]: removed empty attribute

File: html/1woodEnergyConverter.html (hunk 3 above)
Comment (owner6) [reply html/1woodEnergyConverter.html:69]: renamed all variables and identifiers to correct ones

File: html/1woodEnergyConverter.html (hunk 4 above)
Comment (owner6) [reply html/1woodEnergyConverter.html:70]: renamed all variables and identifiers to correct ones

File: html/1woodEnergyConverter.html (hunk 2 above)
Comment (Dever4eg) [reply html/1woodEnergyConverter.html:67]: I can still see `class=""` in the code

File: html/1woodEnergyConverter.html (hunk 3 above)
Comment (Dever4eg) [reply html/1woodEnergyConverter.html:69]: I mean to add click handler like this:
telegram-cloud-photo-size-2-5359599945792995471-y

File: html/1woodEnergyConverter.html (hunk 4 above)
Comment (Dever4eg) [reply html/1woodEnergyConverter.html:70]: But it is still `id="result"`, not `id="wood-heigth-result"`, or smth like that. Did you just forget to push your changes?

**Pull request closed without merging**